README.rst

```diff
@@ -135,6 +135,36 @@ Currently only the Authorization Code authentication flow is supported.
      - ``.with_oidc()``
      -
 
+HTTPS Certificates
+~~~~~~~~~~~~~~~~~~
+
+The ``requests`` library does not use the system certificate store to verify TLS certificates, it uses the
+``certifi`` package. This means that if your service provides a certificate that is not publicly trusted
+there are two options.
+
+pip-system-certs
+================
+
+The ``pip-system-certs`` library patches the certificate loading mechanism for ``requests`` causing it to
+use your system certificate store. This is the most seamless solution, but there are a two issues to be
+aware of:
+
+1. The library does not support every platform that is supported by python, so it may not
+be possible for you to use.
+
+2. The change to ``requests`` affects every package in your environment, including pip. Make sure you are
+using a virtual environment.
+
+.. note::
+  If you are using OIDC authentication and your service provides a self-signed certificate you will need
```

Comment:
I usually see self-signed certificates to be certificates without a CA at all, or at least a CA that is only used to sign a single cert for a single user. In this case, the term 'internally-signed certificate' seems to be used, where the CA is an internal CA.

```suggestion
  If you are using OIDC authentication and your service provides a internally-signed certificate you will need
```